ZupIT/ritchie-cli
# Update AWS secrets syntax + unix smoke tests

### Description

- Update AWS secrets syntax
- Update unix smoke tests

### How to verify it

- Check AWS syntax on Ritchie CLI repo secrets
- Run the Release v3 workflow on Github Actions
---------------------------------------------### This pull request generated the following artifacts.
To test the health and quality of this implementation, download the respective binary for your operating system, unzip and directly run the binary like the examples below.
- **Windows** Download the file: **[rit-windows.zip](https://github.com/ZupIT/ritchie-cli/suites/3780939968/artifacts/93143944)** Unzip to some folder like: `C:\home\user\downloads\pr1040` Access the folder: `cd C:\home\user\downloads\pr1040` Directly call the binary: `.\rit.exe --version` or `.\rit.exe name of formula`  - **Linux** Download the file: **[rit-linux.zip](https://github.com/ZupIT/ritchie-cli/suites/3780939968/artifacts/93143942)** Unzip to some folder like: `/home/user/downloads/pr1040` Access the folder: `cd /home/user/downloads/pr1040` Assign execute permission to binary: `chmod +x ./rit` Directly call the binary: `./rit --version` or `./rit name of formula`  - **MacOS** Download the file: **[rit-macos.zip](https://github.com/ZupIT/ritchie-cli/suites/3780939968/artifacts/93143943)** Unzip to some folder like: `/home/user/downloads/pr1040` Access the folder: `cd /home/user/downloads/pr1040` Assign execute permission to binary: `chmod +x ./rit` Directly call the binary: `./rit --version` or `./rit name of formula`> Generated at Wed Sep 15 2021 20:45:26 GMT+0000 (Coordinated Universal Time)
